src/batch/src/monitor/stats.rs

```diff
@@ -173,88 +67,23 @@ pub type BatchMetricsWithTaskLabels = Arc<BatchMetricsWithTaskLabelsInner>;
 pub struct BatchMetricsWithTaskLabelsInner {
```

Comment:
This struct should be renamed

Reply:
`BatchMetricsWithTaskLabelsInner ` -> `BatchMetricsInner`
`BatchMetricsWithTaskLabels ` -> `BatchMetrics`